library/src/components/Markdown.tsx

```diff
@@ -2,19 +2,17 @@ import React from 'react';
 import DOMPurify from 'dompurify';
 import MarkdownIt from 'markdown-it';
 
-import { bemClasses } from '../helpers';
-
 const markdownIt = new MarkdownIt();
 
-export const Markdown: React.FunctionComponent = ({ children }) => {
+interface Props {
+  children?: string | null;
+}
```

Comment:
You don't need to define `children` prop for component :) Please remove it, or maybe write why you want to do that?